### PR TITLE
Add X2Go failed session solution (if an old session stuck on the server)

### PR DESCRIPTION
**Pull request by hajgato**

### CODE REVIEW

File: intro-HPC/ch_x2go.tex [hunk 1]
```diff
@@ -155,3 +155,13 @@ \section{Resume a previous session}
 This will give you the full login name (like ``\strong{\emph{\loginhost{}}}'' but the hostname in your situation may be slightly different).
 You should set the same name to resume the  session the next time.
 Just add this full hostname into ``login hostname'' section in your X2Go session (see \autoref{subsec:sessions-x2go-optionb}).
+
+\section{Connection faild with X2Go}
```
Comment (boegel) [inline intro-HPC/ch_x2go.tex:159]: typo: `faild` -> `failed`

File: intro-HPC/ch_x2go.tex [hunk 2]
```diff
@@ -155,3 +155,13 @@ \section{Resume a previous session}
...
+\label{sec:conn-fail-x2go}
+
+If you get the error ``\strong{Connection Failed.} session vscXXYYY-123-4567890123\_xyzXFCE\_dp32 terminated'' (or similar),
+It is a possibility that an old X2go session remained on the login node.
```
Comment (boegel) [inline intro-HPC/ch_x2go.tex:163]: `It is possible that`

`X2go` -> `X2Go`

File: intro-HPC/ch_x2go.tex [hunk 3]
```diff
@@ -155,3 +155,13 @@ \section{Resume a previous session}
...
+First, choose a different session type (for example TERMINAL), then start the X2Go session.
+A window would pop up, and you should see that a session is running. Select the session and terminate it.
```
Comment (boegel) [inline intro-HPC/ch_x2go.tex:165]: `would` -> `will`?

File: intro-HPC/ch_x2go.tex [hunk 4]
```diff
@@ -155,3 +155,13 @@ \section{Resume a previous session}
...
+Then finish the session, chose again XFCE session (or whatever you use), then you should have your X2Go session.
```
Comment (boegel) [inline intro-HPC/ch_x2go.tex:166]: `chose` -> `choose`

**Review by boegel — CHANGES_REQUESTED** [review]

File: intro-HPC/ch_x2go.tex (hunk 3 above)
Comment (hajgato) [reply intro-HPC/ch_x2go.tex:165]: if it is the problem

File: intro-HPC/ch_x2go.tex (hunk 1 above)
Comment (hajgato) [reply intro-HPC/ch_x2go.tex]: done

File: intro-HPC/ch_x2go.tex (hunk 2 above)
Comment (hajgato) [reply intro-HPC/ch_x2go.tex]: done

File: intro-HPC/ch_x2go.tex (hunk 4 above)
Comment (hajgato) [reply intro-HPC/ch_x2go.tex]: done

**Pull request merged**